pkg/monitor/cluster/machineconfigpool.go

```diff
@@ -0,0 +1,50 @@
+package cluster
+
+//Copyright (c) Microsoft Corporation.
+//Licensed under the Apache License 2.0.
+
+import (
+	"context"
+
+	v1 "github.com/openshift/machine-config-operator/pkg/apis/machineconfiguration.openshift.io/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var machineConfigPoolConditions = map[v1.MachineConfigPoolConditionType]struct{}{
+	v1.MachineConfigPoolUpdated:        {},
+	v1.MachineConfigPoolUpdating:       {},
+	v1.MachineConfigPoolNodeDegraded:   {},
+	v1.MachineConfigPoolRenderDegraded: {},
+	v1.MachineConfigPoolDegraded:       {},
+}
+
```

Comment:
add machineConfigPoolNotConditions, should just include MachineConfigPoolUpdated